pkg/macaroon/caveat.go

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package macaroon
+
+import (
+	"encoding/binary"
+	"time"
+)
+
+// NewCaveat returns a Caveat with a nonce initialized to the current timestamp
+// in nanoseconds.
+func NewCaveat() Caveat {
+	var buf [8]byte
+	binary.BigEndian.PutUint64(buf[:], uint64(time.Now().UnixNano()))
```

Comment:
I can imagine users not being thrilled about each share token revealing the timestamp when it was made. Can we make this random?

Reply:
Sounds good.

Reply:
#1964

Reply:
I've just updated the doc comment on https://github.com/storj/storj/pull/1965 after the applied changes in #1964